src/Redux/books/books.js

```diff
@@ -0,0 +1,37 @@
+const ADD_BOOK = 'addBook';
+const REMOVE_BOOK = 'removeBook';
+
+let initialCount = 0;
```

Comment:
- [ ] Kindly ensure that you set the initial state to be an empty array of books also.